content/pt-br/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Proxies no Kubernetes
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+Esta página descreve o uso de proxies com Kubernetes. 
+
+
+<!-- body -->
+
+## Proxies
+Existem vários tipos diferentes de proxies que você pode encontrar usando Kubernetes:
+
+
+1.  O [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - executa no pc do usuário ou em um pod
+    - proxies de um host local endereçam para o servidor de API do kubernetes
+    - cliente para o proxi usa HTTP 
```

Comment:
I was searching for some usage of 'proxy' work in Portuguese and looks like we don't have a translation, maybe it's better to use the original word, what do you think?

Reply:
Applied at: 4c41cf406475443a52c5aed42b2f1eced897a472